evm/deploy-contract.js

```diff
@@ -31,6 +31,7 @@ const { addExtendedOptions } = require('./cli-utils');
 async function getConstructorArgs(contractName, config, wallet, options) {
     const args = options.args ? JSON.parse(options.args) : {};
```

Comment:
maybe add a try/catch to print a readable error. easy to get args encoding incorrect. also print the JSON prettified args so user can double check